Through The Breach/sheet.json

```
@@ -0,0 +1,8 @@
+{
+	"html": "ThroughTheBreach.html",
+	"css": "ThroughTheBreachStyle.css",
+	"authors": "Doug Broman",
+	"roll20userid": "1786785",
+	"preview": "ThroughTheBreachPreview.png",
+	"instrutions": "A basic character sheet for the Through The Breach Roleplaying Game by Wyrd Miniatures, LLC"
```

```suggestion
	"instructions": "A basic character sheet for the Through The Breach Roleplaying Game by Wyrd Miniatures, LLC"
```
You had a typo here. Otherwise things are looking good